ssl/t1_trce.c

```diff
@@ -428,6 +428,10 @@ static ssl_trace_tbl ssl_ciphers_tbl[] = {
     {0xCCAD, "TLS_DHE_PSK_WITH_CHACHA20_POLY1305"},
     {0xCCAE, "TLS_RSA_PSK_WITH_CHACHA20_POLY1305"},
     {0x1301, "TLS_AES_128_GCM_SHA256"},
+    {0x1302, "TLS_AES_256_GCM_SHA384"},
```

Comment:
Okay, not for this PR, but I'm gonna gripe.  We have all these ```_TXT_``` names and we're not using them.  And why aren't we using the cipher constants?  Again, not for this PR.